src/index.js

```diff
@@ -1,3 +1,83 @@
 import '@fortawesome/fontawesome-free/js/all.js';
+import Score from './score.js';
 
 import('./style/style.css');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/';
+
+class Game {
+  static displayScore = (userInfo) => {
+    const ul = document.querySelector('.board-items');
+    const li = document.createElement('li');
+    li.innerHTML = `
+    <span>${userInfo.user} : </span><span>${userInfo.score}</span>
+    `;
+    ul.appendChild(li);
+  };
+
+  static addScore = (event) => {
```

Comment:
- [ ] [OPTIONAL] GOOD JOB MAN 🍒🕹👌✅👏👏. It's working well 😉. I give one suggestion to improve your project. Please create some statements to handle input type, because I can somehow add a string as a score 😳. 

**Happy coding ☕**

Reply:
I just handled that in CSS and HTML

Reply:
No problem 🙌 now its working